src/HTSeq/_HTSeq.pyx

```diff
@@ -662,15 +662,19 @@ cdef class Sequence( object ):
          new_name = self.name + "[part]"
       return Sequence( self.seq[ item ], new_name )
    
-   def write_to_fasta_file( self, fasta_file ):
+   def write_to_fasta_file( self, fasta_file, bases_per_line=70 ):
```

Comment:
````
'''Write sequence to file in FASTA format.

Arguments:
    - fasta_file (file handle): destination file
    - characters_per_line (int >=0): if 0, write the whole sequence on a single line. Otherwise, break into several lines if the sequence is long enough.

```